Gruntfile.coffee

```diff
@@ -895,13 +895,18 @@ module.exports = (grunt) ->
 		bootlint:
 			all:
 				options:
+					stoponerror: true
```

Comment:
I need to rebase this back out. This was only there to speed up finding the failures.